src/config/man.md

```diff
@@ -1,19 +1,44 @@
 # Manual Pages
 
-Void packages come with manual pages and the default installation includes the
-[mandoc](https://mandoc.bsd.lv/) manpage toolset.
+Many Void packages come with manual ('man') pages. The default installation
+includes the [mandoc](https://mandoc.bsd.lv/) manpage toolset, via the `mdocml`
+package.
 
-The [man(1)](https://man.voidlinux.org/man.1) command can be used to show manual
-pages.
+The [man(1)](https://man.voidlinux.org/man.1) command can be used to show man
+pages:
 
 ```
-$ man 1 chroot
+$ man chroot
 ```
 
+Every man page belongs to a particular *section*:
+
+- 1: User commands (Programs)
+- 2: System calls
+- 3: Library calls
+- 4: Special files (devices)
+- 5: File formats and configuration files
+- 6: Games
+- 7: Overview, conventions, and miscellaneous
+- 8: System management commands
+
+Refer to [man-pages(7)](https://man.voidlinux.org/man-pages.7) for details.
+
+There are some man pages which have the same name, but are used in different
+contexts, and are thus in a different section. You can specify which one to use
+by including the section number in the call to `man`:
+
+```
+$ man 1 printf
+```
+
+`man` can be configured via the
+[man.conf(5)](https://man.voidlinux.org/man.conf.5) file, `/etc/man.conf`.
```

Comment:
Style question: when the man page specifies where the file should be located, I tend to write something like "can be configured in man.conf(5).", without specifying the file itself. What do you think?

Reply:
Good point; agreed. PR updated.